javaparser-core/src/main/java/com/github/javaparser/TokenRange.java

```diff
@@ -0,0 +1,38 @@
+package com.github.javaparser;
+
+import static com.github.javaparser.utils.Utils.assertNotNull;
+
+/**
+ * The range of tokens covered by this node.
```

Comment:
You know what? I'll leave it in beta for a release or two so we can discuss naming.